src/components/Charts/PriceChart.tsx

```diff
@@ -30,13 +28,35 @@ function getDelta(start: number, current: number) {
   return (isPositive ? '+' : '') + delta.toFixed(2)
 }
 
+export const ChartWrapper = styled.div`
+  position: relative;
+`
+
+export const ChartHeader = styled.div`
+  position: absolute;
+`
+
+export const TokenPrice = styled.span`
+  font-size: 36px;
+  line-height: 44px;
+`
+export const DeltaContainer = styled.div`
+  height: 16px;
+  display: flex;
+  align-items: center;
+`
+const ArrowCell = styled.div`
+  padding-left: 2px;
```

Comment:
technically we don't allow 2px, lmk if we can avoid

Reply:
I copied this over from TokenDetail

Reply:
ah ok